packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/index.md

```diff
@@ -25,33 +25,26 @@ This use case requires the **password** and **phone** factors.
 
 ## Configuration updates
 
-Before you build the self-registration flow in your app, configure the Okta org to accept self-registration with both the password and phone factors by performing the following steps.
+Before you build a sign-in flow with password and phone factors, you need to configure the Okta org to accept both factors in your app:
 
-### Step 1: Set up your Okta org for password factor only use cases
+### Step 1: Set up your Okta org for a password factor
```

Comment:
Same comment as before regarding apparent contradication.

Reply:
I can't change this without re-organizing the main common-org-setup instructions originally done by Robert. These instructions affect all framework use cases.

Reply:
Yup, understood.